### ISSUE TITLE
Wine apps menus can't be opened

### ISSUE THREAD

**Issue by Kommynct** [hyprwm/Hyprland#5354]
### Hyprland Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch HEAD at commit c5e28ebcfe00a510922779b2c568cfa52a317445  (props: bump ver 0.37.1).
Date: Sat Mar 16 14:51:49 2024
Tag: v0.37.1

flags: (if any)


System Information:
System name: Linux
Node name: anarchy-desktop
Release: 6.8.2-zen2-1-zen
Version: #1 ZEN SMP PREEMPT_DYNAMIC Thu, 28 Mar 2024 17:06:20 +0000


GPU information: 
06:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Navi 23 [Radeon RX 6650 XT / 6700S / 6800S] [1002:73ef] (rev c1) (prog-if 00 [VGA controller])


os-release: NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo


plugins:


```

</details>

### Bug or Regression?

Bug

### Description

When I open wine apps, their menu buttons can't be focused

### How to reproduce

Open a wine app and try the menu buttons

**Comment by vaxerski** [comment]
mine can be. Gotta be more specific.

**Comment by Kommynct** [comment]
The app I'm using is the logmein rescue client, I'll investigate if it happens elsewhere but it does work on sway

**Comment by Kommynct** [comment]
fixed with 0.42.0
